README.md

```diff
@@ -420,13 +200,37 @@ The `dosageInstruction` property may contain values with the following propertie
 |timing|optional [Timing](https://hl7.org/fhir/R4B/datatypes.html#timing)|-|When medication should be administered|
 |doseAndRate|list of Element|-|Amount of medication administered|
 |doseAndRate>type|optional [DoseRateType](https://hl7.org/fhir/R4B/codesystem-dose-rate-type.html)|e.g. "calculated", "ordered"||
-|doseAndRate>dose|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|TBD: We should either use "pills" or some weight unit here.|
+|doseAndRate>dose|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|Unit is always "tablet". Value may be 0.5, 1 or 2.|
 |maxDosePerPeriod|optional [Ratio](https://hl7.org/fhir/R4B/datatypes.html#ratio)|-|Upper limit on medication per unit of time|
 |maxDosePerAdministration|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|Upper limit on medication per administration|
 |maxDosePerLifetime|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|Upper limit on medication per unit of time|
 
 There may be up to three intakes per day (morning, mid-day and evening) with either 0.5, 1 or 2 pills. The dosages should always be grouped by medication, i.e. there should not be multiple medication elements concerning the same medication but different dosage instructions. Instead, one medication element shall be used for that medication with multiple dosage instructions.
 
+### patients/$userId$/medicationRecommendations/$medicationRecommendationId$
+
+These are the output values of the recommendation algorithms. Depending on the type, the recommendations should be displayed with different icons and texts.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|currentMedication|optional Reference(FHIRMedicationRequest)|e.g. `{"reference":"patients/123/medicationRequest/2"}`|Reference to the existing medication request, if applicable.|
+|recommendedMedication|optional Reference(FHIRMedication)|e.g. `{"reference":"medications/2"}`|Reference to the recommended medication, if applicable. This should always direct to a medication, not a drug.|
```

Comment:
I mean drug as tablet in this context, e.g. "Bisoprolol 5 MG Oral Tablet" - does this make it more clear?

FHIRMedication can be used for either a SCD or IN/MIN type RxNorm code, meaning it can either just describe an ingredient or a specific drug, possibly even with brand name (which we don't have). This is why I made the distinction, as the client can expect to never get a reference such as  "medications/19484/drugs/854904", but rather only ingredient-type medications such as "medications/19484".

If you have a better way of phrasing this please let me know.

Reply:
Thank you for the context!

I like the differentiation between "medication and dosage" and "ingredient-type medication".